app/lib/webpack/create-chain.js

```diff
@@ -273,7 +306,7 @@ module.exports = function (cfg, configName) {
             ignore: ['.*'],
             ignore: ['.*'].concat(
               // avoid useless files to be copied
-              ['electron', 'proton', 'cordova', 'capacitor'].includes(cfg.ctx.modeName)
+              ['electron', 'tauri', 'cordova', 'capacitor'].includes(cfg.ctx.modeName)
                 ? [ 'icons/*', 'app-logo-128x128.png' ]
```

Comment:
capacitor?

Reply:
its not a big deal, just seemed premature and out of scope - but it'll be needed.